tendrl/gluster_integration/flows/update_brick_status/__init__.py

```diff
@@ -0,0 +1,34 @@
+import etcd
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+
+
+class UpdateBrickStatus(flows.BaseFlow):
+    def run(self):
+        logger.log(
+            "info",
+            NS.publisher_id,
+            {"message": "starting flow for updating brick status"}
+        )
+        super(UpdateBrickStatus, self).run()
+        node = self.parameters.get("Node.fqdn")
+        cluster_id = self.parameters.get("TendrlContext.integration_id")
+        self.update_brick_status(node, cluster_id)
+
+    def update_brick_status(self, node, cluster_id):
+        try:
+            bricks = NS._int.client.read(
```

Comment:
you should use the Brick object instead of raw read/writes to etcd

Reply:
Done